static/src/js/components/TemplateList/TemplateList.jsx

```diff
@@ -124,7 +124,7 @@ const TemplateList = () => {
       dataKey: 'lastModified',
       title: 'Last Modified',
       className: 'col-auto',
-      dataAccessorFn: () => moment().format('MM-DD-YYYY')
+      dataAccessorFn: (cellData) => moment(cellData).format('LLLL')
```

Comment:
Can we define this format as a constant please, that way we can reference it from all of our tables and if we decide to change it, we can change it in one spot.